stdlib/format.ml

```diff
@@ -356,7 +356,7 @@ let format_pp_break state size fits breaks =
 let format_pp_token state size = function
 
   | Pp_text s ->
-    format_pp_text state size s
+    format_string state s
```

Comment:
Why is this change correct? I don't understand the logic of the `size` argument, but I would assume that it is not always equal to `String.length s`?

Reply:
From what I've seen (but  maybe I missed something), the size of a `Pp_text` token is always the length of the string.

Reply:
As a non-expert reviewer I would feel more comfortable if you kept `format_pp_text` here: if you turned out to be wrong, this patch would break Format cases that do not use your new features -- whereas any bugs in the rest of the patch are limited to users of the new feature.

Reply:
Done.